utils/uploadToS3.ts

```diff
@@ -0,0 +1,29 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const uploadToS3 = (filePath: string): void => {
+    aws.config.update({
+        accessKeyId: process.env.AWS_ACCESS_KEY,
+        secretAccessKey: process.env.AWS_SECRET_KEY,
+        region: process.env.AWS_REGION,
+        signatureVersion: "v4",
+    });
+    const s3 = new aws.S3();
+
+    const fileStream = fs.createReadStream(filePath);
+    const uploadParams = {
+        Bucket: process.env.AWS_BUCKET_NAME,
+        Key: path.basename(filePath),
+        Body: fileStream,
+    };
+    s3.upload(uploadParams, function (err, data) {
+        if (err) {
+            console.log("ERROR: ", err);
+        } else {
+            console.log("SUCCESS: ", data.Location);
+        }
+    });
+};
+
+export default uploadToS3;
```

Comment:
can you add an example of how this function can be called

Reply:
see https://github.com/uwblueprint/social-diversity-for-children/pull/11/commits/5652c943c9e3b0fb96cab393fd2dfe0a93275e68